README.md

```diff
@@ -4,7 +4,7 @@
 [![Docs-dev](https://img.shields.io/badge/docs-dev-blue.svg)](https://trixi-framework.github.io/Trixi.jl/dev)
 [![Slack](https://img.shields.io/badge/chat-slack-e01e5a)](https://join.slack.com/t/trixi-framework/shared_invite/zt-sgkc6ppw-6OXJqZAD5SPjBYqLd8MU~g)
 [![Youtube](https://img.shields.io/youtube/channel/views/UCpd92vU2HjjTPup-AIN0pkg?style=social)](https://www.youtube.com/@trixi-framework)
-[![Build Status](https://github.com/trixi-framework/Trixi.jl/workflows/CI/badge.svg)](https://github.com/trixi-framework/Trixi.jl/actions?query=workflow%3ACI)
+[![Build Status](https://github.com/trixi-framework/Trixi.jl/actions/workflows/ci.yml/badge.svg)](https://github.com/trixi-framework/Trixi.jl/actions?query=workflow%3ACI)
```

Comment:
Nice idea! In this case, we should also use the `main` branch

```suggestion
[![Build Status](https://github.com/trixi-framework/Trixi.jl/actions/workflows/ci.yml/badge.svg?branch=main)](https://github.com/trixi-framework/Trixi.jl/actions?query=workflow%3ACI)
```

Reply:
Yes, we could do this!

However, it seems as if the main branch is picked by default anyway, although this is not mentioned.

See TrixiAtmo.jl status without branch:  <img src="https://github.com/trixi-framework/TrixiAtmo.jl/actions/workflows/ci.yml/badge.svg">
and with main branch: <img src="https://github.com/trixi-framework/TrixiAtmo.jl/actions/workflows/ci.yml/badge.svg?branch=main">

The latest CI run for TrixiAtmo.jl however failed.

Reply:
Ok, thanks. If that's the case, we don't need to make it more complex than necessary. Thanks